core/prelude/types/uint.carbon

```diff
@@ -32,6 +33,21 @@ impl forall [N:! IntLiteral()] UInt(N) as As(IntLiteral()) {
   fn Convert[self: Self]() -> IntLiteral() = "int.convert_checked";
 }
 
+// TODO: Allow as an implicit conversion if N > M.
+impl forall [M:! IntLiteral(), N:! IntLiteral()] UInt(M) as As(UInt(N)) {
+  fn Convert[self: Self]() -> UInt(N) = "int.convert";
+}
+
+// TODO: Allow as an implicit conversion if N > M.
```

Comment:
N > M + 1?

Reply:
I think this is right as-is: all values of uN fit in i(N+1), so it's OK for that to be implicit. The *other* comments could use `>=` instead of `>`, but that would only allow identity conversions which we already allow.

Reply:
Ah, sorry, I for some reason read this as `>=` instead of `>`.